docs/ccga_dx.md

```diff
@@ -0,0 +1,9 @@
+# nf-core/configs: CCGA DX Configuration
+
+Deployment and testing of nf-core pipelines at the CCGA DX cluster is on-going. 
+
+To use, run the pipeline with `-profile ccga_dx`. This will download and launch the [`ccga.config`](../conf/ccga.config) which has been pre-configured with a setup suitable for the CCGA cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
```

Comment:
```suggestion
To use, run the pipeline with `-profile ccga_dx`. This will download and launch the [`ccga_dx.config`](../conf/ccga_dx.config) which has been pre-configured with a setup suitable for the CCGA cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
```

Reply:
Do you think its also worth adding some description regarding the iGenomes source you have available locally?
https://github.com/nf-core/configs/blob/master/docs/crick.md

Reply:
Could also add the same to `ccga.md`

Reply:
In principle, this is a fair point, in practice this particular cluster is my "personal playground". I can maybe add it to the ccga.config at some point, which is used by several people.

Reply:
Ok. Fair point. Would be good if you add it in when you know where its going to reside 👍